refinery_core/src/util.rs

```diff
@@ -7,6 +7,11 @@ use std::path::{Path, PathBuf};
 use std::sync::OnceLock;
 use walkdir::{DirEntry, WalkDir};
 
+#[cfg(not(feature = "int8-versions"))]
+pub type SchemaVersion = i32;
+#[cfg(feature = "int8-versions")]
+pub type SchemaVersion = i64;
+
```

Comment:
features in Rust should be additive not mutually exclusive, enabling `--all-features` should not break compilation, see [here](https://doc.rust-lang.org/cargo/reference/features.html#feature-unification)

Reply:
Enabling `--all-features` doesn't, as far as I'm aware, break compilation.  CI is all green.